quarkus/admin/build.gradle.kts

```diff
@@ -84,3 +85,17 @@ tasks.named("sourcesJar") { dependsOn("compileQuarkusGeneratedSourcesJava") }
 tasks.named("javadoc") { dependsOn("jandex") }
 
 tasks.named("quarkusDependenciesBuild") { dependsOn("jandex") }
+
+tasks.named("distZip") { dependsOn("quarkusBuild") }
+
+tasks.named("distTar") { dependsOn("quarkusBuild") }
+
+distributions {
+  main {
+    contents {
+      from("../../NOTICE")
+      from("../../LICENSE-BINARY-DIST").rename("LICENSE-BINARY-DIST", "LICENSE")
```

Comment:
`LICENSE-BINARY-DIST` includes more than the admin tool uses by itself... but I guess that's ok since it is part of the Polaris project as a whole.